.husky/.gitignore

```diff
@@ -0,0 +1 @@
+_
```

Comment:
nit: this file can be removed as per changelog in https://github.com/typicode/husky/releases/tag/v7.0.0

Reply:
Removed!